splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/TriggerRowHolderImpl.java

```diff
@@ -265,6 +292,8 @@ public void readExternal(ObjectInput in) throws IOException, ClassNotFoundExcept
         CID = in.readLong();
         conglomCreated = in.readBoolean();
         resultDescription = (ResultDescription) in.readObject();
+        needsTemporaryConglomerate = in.readBoolean();
+        needsTemporaryConglomerateSet = in.readBoolean();
```

Comment:
This is just to make sure any decision made in the OlapServer about needing the temporary conglomerate would get reflecting in any execution happening in the executors.  It may not be needed at all, but is there for consistency.

Reply:
I see. Thanks for explaining!